src/main/java/com/google/api/generator/engine/ast/JavaDocComment.java

```diff
@@ -115,6 +115,11 @@ public JavaDocComment build() {
       if (!Strings.isNullOrEmpty(deprecated)) {
         componentsList.add(String.format("@deprecated %s", deprecated));
       }
+      // Escape component in list one by one, because we will join the components by `\n`
+      // `\n` will be taken as escape character by the comment escaper.
+      for (int i = 0; i < componentsList.size(); i++) {
+        componentsList.set(i, CommentEscaper.escape(componentsList.get(i)));
```

Comment:
Inline with `stream()`?

Reply:
sure, TWIL `list.stream().forEach()` only iterates the values but cannot change them, while `list.stream().map` can modify the value but the return value of the method must be used (reassigning to `componentList`).